### PR TITLE
XW-3618 | MCFooter layout

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3618
https://wikia-inc.atlassian.net/browse/XW-3621

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Recirculation/styles/mixed-content-footer.scss [hunk 1]
```diff
@@ -3,13 +3,126 @@
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-functions/index';
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-mixins/index';
 
+$mcf-padding-horizontal: 10px;
+$mcf-padding-vertical: 12px;
+$mcf-proportion: 0.874251497;
```
Comment (rogatty) [inline extensions/wikia/Recirculation/styles/mixed-content-footer.scss]: Let's compute this value to make it more understandable.

**Review by rogatty — APPROVED** [review]

File: extensions/wikia/Recirculation/styles/mixed-content-footer.scss [hunk 2]
```diff
@@ -24,38 +137,51 @@
 			color: $wds-color-link;
 		}
 	}
+
+	@media #{$wds-breakpoint-xxl-up} {
+		width: $mcf-card-width-xxl;
```
Comment (hakubo) [inline extensions/wikia/Recirculation/styles/mixed-content-footer.scss:118]: we used to have a mixin 'square' is it still available?

File: extensions/wikia/Recirculation/templates/Recirculation_article.mustache [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<a href="#" class="mcf-card mcf-card-wiki-article"
+   style="background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000), url(https://unsplash.it/400);">
+			<span class="mcf-card-wiki-article__title">
+				Everything You Need to Know About ‘Iron Fist’ (1:36)
+			</span>
+	<span class="mcf-card-wiki-article__subtitle">
+				Marvel Wiki
+			</span>
+</a>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_article.mustache]: \n

File: extensions/wikia/Recirculation/templates/Recirculation_exploreWikis.mustache [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<div class="mcf-card mcf-card-related-wikis">
+	<header class="mcf-card-related-wikis__header">
+		Explore Wikis
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_exploreWikis.mustache]: i18n

File: extensions/wikia/Recirculation/templates/Recirculation_exploreWikis.mustache [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+			<a href="#" class="mcf-card-related-wikis__item-link">
+				<img src="http://via.placeholder.com/100x56" class="mcf-card-related-wikis__item-image">
+				<div>
+					<span class="mcf-card-related-wikis__title">Twin Peaks Wiki</span>
+					<span class="mcf-card-related-wikis__subtitle">240 recent edits</span>
+				</div>
+			</a>
+		</li>
+	</ul>
+</div>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_exploreWikis.mustache]: \n

File: extensions/wikia/Recirculation/templates/Recirculation_storyStream.mustache [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<a href="#" class="mcf-card mcf-card-ns-stream" style="background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000), url(https://unsplash.it/400);">
+		<span class="mcf-card-ns-stream__title">
+			HBO hit show “Girls” Continues It Dominance
+		</span>
+	<div class="wds-button wds-is-squished wds-is-secondary">Explore posts</div>
+</a>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_storyStream.mustache]: \n

File: extensions/wikia/Recirculation/templates/Recirculation_storyStream.mustache [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<a href="#" class="mcf-card mcf-card-ns-stream" style="background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000), url(https://unsplash.it/400);">
+		<span class="mcf-card-ns-stream__title">
+			HBO hit show “Girls” Continues It Dominance
+		</span>
+	<div class="wds-button wds-is-squished wds-is-secondary">Explore posts</div>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_storyStream.mustache]: i18n

File: extensions/wikia/Recirculation/templates/Recirculation_topic.mustache [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<a href="#" class="mcf-card mcf-card-topic" style="background-image: url(https://unsplash.it/400);">
+	<div class="mcf-card-topic__overlay">
+			<span class="mcf-card-topic__title">
+				Games
+			</span>
+		<div class="wds-button wds-is-squished wds-is-secondary">Explore</div>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_topic.mustache]: i18n

File: extensions/wikia/Recirculation/templates/Recirculation_topic.mustache [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+	</div>
+</a>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_topic.mustache]: \n

File: extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache [hunk 10]
```diff
@@ -0,0 +1,25 @@
+<div class="mcf-card mcf-card-wiki-articles">
+	<header class="mcf-card-wiki-articles__header"{{#communityHeaderBackground}} style="background-image: url({{communityHeaderBackground}});"{{/communityHeaderBackground}}>
+	<span class="mcf-card-wiki-articles__header-text">More {{sitename}}</span>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache]: i18n

File: extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+	</header>
+	<ul class="mcf-card-wiki-articles__list">
+		<li class="mcf-card-wiki-articles__item">
+			<a href="#" class="mcf-card-wiki-articles__item-link">
+				<span class="mcf-card-wiki-articles__circle">1</span>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache:8]: could this be done with css counters?
https://stackoverflow.com/questions/23610151/can-you-style-ordered-list-numbers

File: extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+			</a>
+		</li>
+		<li class="mcf-card-wiki-articles__item">
+			<a href="#" class="mcf-card-wiki-articles__item-link">
+				<span class="mcf-card-wiki-articles__circle">3</span>
+				<span class="mcf-card-wiki-articles__title">Stephen Strange (Earth-616)</span>
+			</a>
+		</li>
+	</ul>
+</div>
```
Comment (hakubo) [inline extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache]: \n

**Review by hakubo — CHANGES_REQUESTED** [review]

File: extensions/wikia/Recirculation/styles/mixed-content-footer.scss (hunk 2 above)
Comment (latata) [reply extensions/wikia/Recirculation/styles/mixed-content-footer.scss:118]: but that's not a square ;)

File: extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache (hunk 11 above)
Comment (latata) [reply extensions/wikia/Recirculation/templates/Recirculation_wikiArticles.mustache:8]: I think it's not worth doing it and it probably won't work for our purpose.

**Review by hakubo — APPROVED** [review]

**Pull request merged**